src/AutoMapper/MapperConfiguration.cs

```diff
@@ -86,7 +87,9 @@ public void Validate(ValidationContext context)
 
         public IEnumerable<IExpressionResultConverter> ResultConverters { get; }
 
-        public IEnumerable<IExpressionBinder> Binders { get; }
+        public IEnumerable<IExpressionBinder> Binders { get; }
+        
+        public int RecursiveQueriesMaxDepth { get; }
```

Comment:
A little late perhaps, but maybe we should move the `ProjectTo` configuration to an inner object? I think this stuff doesn't even warrant a major version bump. Just sayin' :)